test/ussd_nurse.test.js

```diff
@@ -230,7 +235,11 @@ describe("app", function() {
                             ].join('\n')
                         })
                         .check(function(api) {
-                            utils.check_fixtures_used(api, [62, 77, 85]);
+                            // console.log(api.http.fixtures.fixtures[20].request.headers);
+                            // maybe create new function that list table of fixture keys/descriptions against number
+                            // utils.list_fixtures  (used vs unused)
+                            utils.check_fixtures_used(api, [50, 100, 103]);
+                            // check_fixtures_used(api, [sbm_2, sbm_4, is_1]);  // or adapt to something like this...?
```

Comment:
I wouldn't bother - just change the fixture comment numbers to align with these numbers.  So sbm fixtures to start at 50 and is fixtures to start at 100.